README.md

```diff
@@ -271,3 +274,33 @@ ENTRYPOINT ["/go-binary"]
 ### Build final docker image
 
 `docker build -t docker-edgex-binary -f Dockerfile --build-arg BASE=base-build .`
+
+## Deployment On Sandbox
+
+The official site for Sandbox environment is **[https://jenkins.edgexfoundry.org/sandbox/](https://jenkins.edgexfoundry.org/sandbox/)**
+
+#### Get Access To Sandbox
+
+The official process to request for access to sandbox is provided in the link **[Here](https://https://lf-releng-docs.readthedocs.io/en/latest/jenkins-sandbox.html#get-access-to-the-jenkins-sandbox)**
+
+If you already have the LFID and can access the Prod Environment you can use the same LFID to access the Sandbox.
```

Comment:
I don't think this line should be here. The site linked above contains the needed info, and this is misleading. Yes, you'll use the same LFID to access the sandbox as prod, but this seems to imply that if you have access to prod, you immediately have sandbox access, which is not true. As the docs show, each user has to request access to the sandbox from LF.

Reply:
Updated.